framework/include/base/Assembly.h

```diff
@@ -1,16 +1,13 @@
-/****************************************************************/
-/*               DO NOT MODIFY THIS HEADER                      */
-/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
-/*                                                              */
-/*           (c) 2010 Battelle Energy Alliance, LLC             */
-/*                   ALL RIGHTS RESERVED                        */
-/*                                                              */
-/*          Prepared by Battelle Energy Alliance, LLC           */
-/*            Under Contract No. DE-AC07-05ID14517              */
-/*            With the U. S. Department of Energy               */
-/*                                                              */
-/*            See COPYRIGHT for full restrictions               */
-/****************************************************************/
+/**************************************************************************************************/
+/*                                DO NOT MODIFY THIS HEADER                                       */
+/*                  MOOSE - Multiphysics Object Oriented Simulation Environment                   */
+/*                                                                                                */
+/*                    Copyrights reserved, please see COPYRIGHT for details                       */
+/*                    https://github.com/idaholab/moose/blob/master/COPYRIGHT                     */
+/*                                                                                                */
+/*                    Licensed under LGPL 2.1, please see LICENSE for details                     */
+/*                    https://www.gnu.org/licenses/lgpl-2.1.html                                  */
```

Comment:
We actually do not need these two links, they should both are included in COPYRIGHT and LICENSE files.

Reply:
Except that these are here specifically for the case when files are separated from the repository (i.e. somebody mails a file to somebody else).

Reply:
oh. One more thing, is our LICENSE file the same as https://www.gnu.org/licenses/lgpl-2.1.html?